testing/certificates/client.pem

```diff
@@ -0,0 +1,111 @@
+Certificate:
```

Comment:
This file is not used by tests, but it’s part of the setup described in the README. It should either be included or in the .gitignore list, no strong preference, for now I just decided to include it.